test/unit/utils-deserialise-test.js

```diff
@@ -54,3 +58,21 @@ test('accounts with profile response', function (t) {
 
   t.end()
 })
+
+test('accounts with complex relationships', function (t) {
+  var data = deserialise(complexResponse, {
+    include: 'pet.vet'
+  })
+
+  t.deepEqual(data, complexReturn, 'returns right data')
+
+  t.end()
+})
+
+test('accounts with missing includes', function (t) {
+  var data = deserialise(accounts, options)
+
+  t.deepEqual(data, accountsReturn, 'returns right data')
+
+  t.end()
+})
```

Comment:
for the complex relationship, instead of having account -> profile -> vet, can you do session -> account -> profile? You can take this as a server response fixture: https://github.com/hoodiehq/hoodie-server-account/blob/master/tests/integration/fixtures/session-with-profile-response.json

The rest looks great, thanks for the enormous efforts Chris! :bouquet: